### PR TITLE
add new tools to compute how much cloud units is used by a workload

### LINKED ISSUE #1423: Creation of a tool to compute how much cloud unit a workload consumes
It would be useful to have a library that allows user to know how much a workload will consume in term of cloud units.
This allow user to know how much capacity he needs to have in its pool before provisioning the workloads.

The code to compute how much resource units a workload comsume is available in the explorer:
- container: https://github.com/threefoldtech/tfexplorer/blob/046188aa3f9383fdb5e17f420e6d62f5fb4ba18a/models/generated/workloads/container.go#L119
- k8s: https://github.com/threefoldtech/tfexplorer/blob/master/models/generated/workloads/k8s.go#L24
volume: https://github.com/threefoldtech/tfexplorer/blob/master/models/generated/workloads/volume.go#L18
- zdb: https://github.com/threefoldtech/tfexplorer/blob/master/models/generated/workloads/zdb.go#L22

### PR DESCRIPTION
**Pull request by zaibon**
### Description

Add a new `consumption` section in `j.tools.zos` which allow to compute how much cloud units are consumed by a workload object

### Changes

- Add new module in j.tools.zos
- Add `resource_units` methods on all the workloads model

### Related Issues

fixes #1423

### Checklist

- [x] Tests included
- [x] Build pass
- [x] Documentation
- [x] Code format and docstrings

### CODE REVIEW

File: jumpscale/clients/explorer/test_models.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
+from .models import DiskType, Container, Volume, ZdbNamespace, K8s
```
Comment (xmonader) [inline jumpscale/clients/explorer/test_models.py:1]: can we move the tests into tests/clients/explorer instead

File: jumpscale/clients/explorer/models.py [hunk 2]
```diff
@@ -412,6 +471,19 @@ class Container(Base):
     capacity = fields.Object(ContainerCapacity)
     info = fields.Object(ReservationInfo)
 
+    def resource_units(self):
+        cap = self.capacity
+        resource_units = ResourceUnitAmount()
+        resource_units.cru = cap.cpu
+        resource_units.mru = round(cap.memory / 1024)
```
Comment (LeeSmet) [inline jumpscale/clients/explorer/models.py:494]: mru is a float so rounding here only makes sense if we want to limit the amount of significant digits like it is done on the next line imo

**Review by LeeSmet — CHANGES_REQUESTED** [review]
I don't really like the idea of giving 50GB of SSD for containers for free, but it is what it is I guess

**Review by LeeSmet — APPROVED** [review]
lgtm

**Review by xmonader — APPROVED** [review]

**Review by xmonader — APPROVED** [review]

**Pull request merged**